File: app/src/main/java/com/gigabytedevelopersinc/app/CometOTP/Activities/LicensesActivity.java
/*
 * Copyright (C) 2018 Gigabyte Developers
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in all
 * copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
 */

package com.gigabytedevelopersinc.app.CometOTP.Activities;

import android.os.Bundle;
import android.support.v7.widget.Toolbar;
import android.webkit.WebView;

import com.gigabytedevelopersinc.app.CometOTP.R;

/**
 * @author Created by Emmanuel Nwokoma (Founder and CEO at Gigabyte Developers) on 4/10/2018
 **/
public class LicensesActivity extends BaseActivity {

    //private WebView mWebView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setTitle(R.string.licenses_activity_title);
        setContentView(R.layout.licenses_activity);

        Toolbar toolbar = findViewById(R.id.container_toolbar);
        setSupportActionBar(toolbar);

        WebView webViewer = findViewById(R.id.web_view);
        webViewer.loadUrl("file:///android_asset/www/licenses.html");
    }
}

File: app/src/main/java/com/gigabytedevelopersinc/app/cometOTP/Activities/LicensesActivity.kt
@file:Suppress("PackageName")
package com.gigabytedevelopersinc.app.cometOTP.Activities

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.ViewStub
import android.webkit.WebView
import androidx.appcompat.widget.Toolbar
import com.gigabytedevelopersinc.app.cometOTP.R

/**
 * @author Created by Emmanuel Nwokoma (Founder and CEO at Gigabyte Developers) on 4/10/2018
 **/
class LicensesActivity : BaseActivity() {

    //private WebView mWebView;

    @SuppressLint("MissingInflatedId")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setTitle(R.string.licenses_activity_title)
        setContentView(R.layout.activity_container)

        val toolbar = findViewById<Toolbar>(R.id.container_toolbar)
        setSupportActionBar(toolbar)

        val stub = findViewById<ViewStub>(R.id.container_stub)
        stub.layoutResource = R.layout.licenses_activity
        stub.inflate()

        val webViewer = findViewById<WebView>(R.id.web_view)
        webViewer.loadUrl("file:///android_asset/www/licenses.html")
    }
}
